components/data_comps/docn/src/docn_comp_mod.F90

```diff
@@ -1174,6 +1175,55 @@ subroutine prescribed_sst(xc, yc, lsize, sst_option, sst)
        end do
     end if
 
+    !-------------------------------------------------------------------------------
```

Comment:
Nitpicky, but this seems like a lot of unnecessary copy and pasting of code that could be easily condensed.

Reply:
Yea, good point - I'll clean it up.

Reply:
@bogensch can you take a look at how I updated this block?